AppBundles/ExportDrawingAsPdfPlugin/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
```

Comment:
https://github.com/Developer-Autodesk/forge-configurator-inventor/wiki/How-to-add-new-Inventor-plugin#use-shared-version-info